Sources/Imperial/Helpers/Sessions+Imperial.swift

```diff
@@ -15,17 +15,29 @@ extension Request {
 }
 
 extension Session {
-    
+	
+	enum Keys {
```

Comment:
Nitpick: The indentation whitespace for this enum should match the indentation for the rest of the extensions methods. Also, the whitespace above it should be removed to it matches the rest of the extensions and types in the project.